app/experimenter/experiments/models.py

```diff
@@ -256,6 +256,12 @@ class Filters:
                 status=NimbusConstants.Status.DRAFT,
                 publish_status=NimbusConstants.PublishStatus.APPROVED,
             )
+            | Q(
+                is_rollout=True,
+                status=NimbusConstants.Status.LIVE,
+                status_next=NimbusConstants.Status.LIVE,
+                publish_status=NimbusConstants.PublishStatus.APPROVED,
```

Comment:
If we're adding this here, shouldn't there be a test below to test the case that we do reallocate buckets for a live update?

Reply:
It's below in `test_models`